source/_integrations/tuya.markdown

```diff
@@ -48,19 +48,20 @@ You have created an account for [Tuya Smart or Smart Life app](https://developer
 
 **1.** Log in to the [Tuya IoT Platform](https://iot.tuya.com/).
```

Comment:
Add a note here that this portal requires a separate account from your app.

Reply:
Added